student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(id, age, _classroomm, name: 'Unknown', parent_permission: true)
+    super(id, age, name: name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Great job on your classes 👏. However, there seems to be a typo in the `initialize` method of the `Student` class. The parameter for classroom is misspelled as `_classroomm`, and it should be corrected to `classroom`.  Please ensure to correct that; otherwise, you will get errors when you try to use this class.  👌

```suggestion
  def initialize(id, age, classroom, name: 'Unknown', parent_permission: true)
    super(id, age, name: name, parent_permission: parent_permission)
    @classroom = classroom
  end
```